deployments/k8s/configmap.yaml

```diff
@@ -144,6 +144,7 @@ data:
       logLevel: info
 
     metricsToExclude:
+      - {"#from": "/etc/signalfx/whitelist.yaml", flatten: true}
       - metricNames:
```

Comment:
You can probably remove these filters since the whitelist probably already covers them (confirm though).